queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
- [ ]  `LIKE '%mon%'` means Finds any values that have "mon" in any position, and According to the requirements we need to Find all animals whose name ends in "mon"., So it would be better to use `LIKE '%mon'` 💯